gradle/libs.versions.toml

```diff
@@ -67,6 +72,23 @@ kotlinx-io-reactor-extensions = { module = "io.projectreactor.kotlin:reactor-kot
 kotlinx-serialization-json = { module = "org.jetbrains.kotlinx:kotlinx-serialization-json-jvm", version.ref = "kotlinx-serialization" }
 kotlinx-serialization-protobuf = { module = "org.jetbrains.kotlinx:kotlinx-serialization-protobuf", version.ref = "kotlinx-serialization" }
 
+# Micronaut
+snakeyaml = { module = "org.yaml:snakeyaml", version.ref = "snakeyaml" }
+micronaut-parent = { module = "io.micronaut.platform:micronaut-parent", version = "4.7.1" }
```

Comment:
version misalignments, above 4.4.3, but here 4.7.1, all micronaut packages share the same version.